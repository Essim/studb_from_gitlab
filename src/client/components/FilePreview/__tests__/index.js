import React from 'react';
import {FilePreview} from '../';
import {shallowToJson} from 'enzyme-to-json';

const mockDoc_cuid = "mockCuid",
    mockFile = {
        name: 'Nouveau document',
        path: 'http://localhost:8888/files/cjams2twk0000w3ffq8frjxu7',
        type: 'application/octet-stream'
    };

describe('FilePreview', () => {
    it('should render correctly', () => {
        const output = shallow(<FilePreview doc_cuid={mockDoc_cuid} selected_file={mockFile}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
