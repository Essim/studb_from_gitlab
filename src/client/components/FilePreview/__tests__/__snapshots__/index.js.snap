// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilePreview should render correctly 1`] = `
<div>
  <Component
    allowFullScreen={false}
    loadingMsg="loading..."
    scrolling="auto"
    showLoadingMsg={false}
    src="http://localhost:8888/files/cjams2twk0000w3ffq8frjxu7"
  />
</div>
`;
